cms/templates/settings.html

```diff
@@ -328,13 +333,11 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <li class="field text" id="field-course-overview">
                   <label for="course-overview">${_("Course Overview")}</label>
                   <textarea class="tinymce text-editor" id="course-overview"></textarea>
-                  <%def name='overview_text()'><%
-                    a_link_start = '<a class="link-courseURL" rel="external" href="'
-                    a_link_end = '">' + _("your course summary page") + '</a>'
-                    a_link = a_link_start + lms_link_for_about_page + a_link_end
-                    text = _("Introductions, prerequisites, FAQs that are used on %s (formatted in HTML)") % a_link
-                    %>${text}</%def>
-                  <span class="tip tip-stacked">${overview_text()}</span>
+                  <span class="tip tip-stacked">${
```

Comment:
@robrap This is the section that was not caught previously in #12042. But it just resulted in HTML being improperly displayed as text, as opposed to a possible injection.